src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+export const NewCommentForm = ({ id, onAddComment }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
+
+  const handleSubmit = () => {
+    onAddComment(id, name, email, body);
+
+    setName('');
+    setEmail('');
+    setBody('');
+  };
+
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={(event) => {
+        event.preventDefault();
+        handleSubmit();
+      }}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          type="text"
+          value={name}
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          onChange={event => setName(event.target.value.trimLeft())}
```

Comment:
```suggestion
          onChange={({ target }) => setCommentPart(target.name, target.value)}
```
create method setCommentPart which should work like this:
```javascript
const setCommentPart = (key, value) => {
  setComment(prevComment => ({
    ...prevComment,
    [key]: value,  
  })
}
```

Reply:
https://prnt.sc/ukm6jw
Тому залишу як було)